src/components/skillformular.tsx

```diff
@@ -1,5 +1,6 @@
 import React, { useState } from "react";
 import { Link } from "react-router-dom";
+import "./bootstrapSetup";
```

Comment:
das Problem ist leider immernoch dasselbe. Man müsste jetzt für jede einzelne .tsx file im Projekt das Bootstrap importieren.
Probier mal bootstrap nur in der übergeordneten index.js zu importieren und sonst nirgends. Dann müsste der Style global verfügbar sein.